news/2024/2024-11-25-the-followpoint-hivie-the-man-of-anything-and-everything.md

```diff
@@ -0,0 +1,360 @@
+---
+layout: post
+title: "The Followpoint: Hivie, the Man of Anything and Everything"
```

Comment:
I kinda agree it reads wrong.. "Ever heard of the person who does literally anything you can think of?"

hivie does a great job around the place, but i wouldn't go as far as saying "literally anything and everything", especially when only four roles are touched on.

i'm not trying to downplay their role, but it may read a bit odd to others who are involved in a similar number of community ventures.

Reply:
Tbh yea, I admit I went clickbait mode here, so if you have any good ideas - I'll gladly change the title. But like I said, the idea is to tell people Hivie does a lot, so I will not agree to something like `osu!taiko Paragon`

Reply:
@Walavouchey @peppy What do you think of "Hivie, the Powerhouse Behind the Scenes"?

Reply:
Dunno. @Ephemeralis might be able to give a suiting one here since he previously had kinda the same role.